test/old/testdir/test_registers.vim

```diff
@@ -838,5 +838,76 @@ func Test_replay_charsearch_omap()
   bwipe!
 endfunc
 
+" Make sure that y_append is correctly reset
```

Comment:
Drive-by review: Should the comment change given that this variable apparently no longer exists in Neovim according to the PR description?

Reply:
No